acestep/core/generation/handler/generate_music_execute.py

```
@@ -25,12 +34,55 @@ def _run_generate_music_service_with_progress(
+                    audio_cover_strength=audio_cover_strength,
+                    cover_noise_strength=cover_noise_strength,
+                    use_adg=use_adg,
+                    cfg_interval_start=cfg_interval_start,
+                    cfg_interval_end=cfg_interval_end,
+                    shift=shift,
+                    infer_method=infer_method,
+                    audio_code_hints=service_inputs["audio_code_hints_batch"],
+                    return_intermediate=service_inputs["should_return_intermediate"],
+                    timesteps=timesteps,
+                )
+            except Exception as exc:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)